.buildkite/scripts/dockerhub_upload.sh

```diff
@@ -4,14 +4,30 @@ set -euo pipefail
 
 source .buildkite/scripts/shared.sh
 version=$(buildkite-agent meta-data get "version")
+channel=$(buildkite-agent meta-data get "release-channel")
 
 if is_fake_release; then
   # This overrides IMAGE_NAME in docker-base.sh
   export IMAGE_NAME="habitat/fakey-mc-fake-face-studio"
+else
+  export IMAGE_NAME="habitat/default-studio"
```

Comment:
Same w/r/t exporting and upper/lower case, and the comment about `docker-base.sh`